docs/examples/src/test/java/generic/ImagePullPolicyTest.java

```diff
@@ -0,0 +1,18 @@
+package generic;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.image.pull.policy.PullPolicy;
+
+public class ImagePullPolicyTest {
```

Comment:
This test never runs. Please verify that it works correctly.
You could also refer existing tests from `core/src/test/java` to avoid a duplication